packages/app/package.json

```diff
@@ -37,6 +37,7 @@
     "@roadiehq/backstage-plugin-github-insights": "^0.2.7",
     "@roadiehq/backstage-plugin-github-pull-requests": "^0.5.2",
     "@roadiehq/backstage-plugin-travis-ci": "^0.2.3",
+    "@roadiehq/backstage-plugin-catalog-import": "^0.1.0",
```

Comment:
As we discussed, since this will be part of the core experience it needs to be part of the main repo. Ping @dtuite

Reply:
Hey Stefan. I dropped off the call before this was discussed. I will send you a message on Discord.